modules/python/init.zsh

```diff
@@ -147,12 +147,16 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) && \
 fi
 
 # Load PIP completion.
-if (( $#commands[(i)pip(|[23])] )); then
+for pip_command in pip{,2,3}; do
+  if [[ "$PYENV_ROOT" ]]; then
+    [[ "$(pyenv which "$pip_command" 2>/dev/null)" ]] && break
+  else
+    [[ "$(which "$pip_command" 2>/dev/null)" ]] && break
```

Comment:
Using `$command` is nice, but using `which` gives it a nice symmetry.

Reply:
I like the symmetry as well, but would want some more comments on the change before accepting.

Reply:
Also, there is a downside to using `which`. If any of these are overridden to be functions or aliases, the cache check below `if [[ "$pip_command" -nt "$cache_file" ...]]` will fail.

Reply:
> Also, there is a downside to using which. If any of these are overridden to be functions or aliases, the cache check below if [[ "$pip_command" -nt "$cache_file" ...]] will fail.

Interestingly, the result of calling `which` is never used. So whether or not it is an alias, `[[ "$pip_command" -nt "$cache_file" ...]]` _always_ fails! This is an even bigger bug.